iwyu.cc

```diff
@@ -898,21 +898,13 @@ class BaseAstVisitor : public RecursiveASTVisitor<Derived> {
         const_cast<CXXDestructorDecl*>(dtor), parent_type, expr);
   }
 
-  // This is to catch assigning template functions to function pointers.
+  // This is to catch binding of function pointers to templates.
```

Comment:
Done, toned down binding and added test cases for simple, "naked" function pointer expressions.